go.mod

```diff
@@ -61,6 +61,14 @@ require (
 	golang.org/x/text v0.3.6 // indirect
 	golang.org/x/tools v0.1.3-0.20210608163600-9ed039809d4c // indirect
 	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 // indirect
-	google.golang.org/protobuf v1.25.0 // indirect
+	google.golang.org/protobuf v1.26.0-rc.1 // indirect
 	honnef.co/go/tools v0.2.0 // indirect
 )
+
+require (
```

Comment:
you are likely not running go 1.17.2, please upgrade your go to 1.17.2, which fixed this bug.

Reply:
yea I found out that this bug is still not fixed in go 1.17.2 :sigh:

you can just manually remove line 66-68 (so merge the two indirect require blocks), and rerun `go mod tidy` so it sorts that block correctly for you.